src/components/Form/SetAclPermissionForm.jsx

```diff
@@ -43,14 +42,9 @@ const SetAclPermissionForm = () => {
     setDocType(event.target.value);
   };
 
-  const clearInputFields = () => {
-    dispatch({ type: 'CLEAR_PROCESSING' });
-  };
-
   // Event handler for setting ACL permissions to file container on Solid
   const handleAclPermission = async (event) => {
     event.preventDefault();
-    dispatch({ type: 'SET_PROCESSING' });
```

Comment:
The original intent for `dispatch({ type: 'SET_PROCESSING' });` was to set a state for the button in use for these forms/modals to be disabled so that it'll prevent additional clicks for that button when the function is running.

Could you set up a new useState in place of this (i.e. `processing` remains true until it reaches the finally block where the setTimeout is located, so something like `setProcessing(true)`?)

The same would be applied for all the other dispatch calls being removed.